packages/core/contracts/optimistic-asserter/implementation/escalation-manager/SuperbondEscalationManager.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity 0.8.16;
+
+import "./BaseEscalationManager.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "../../interfaces/OptimisticAsserterInterface.sol";
+
+// This EscalationManager allows to arbitrate for each assertion based on its bond via the DVM or the EscalationManager.
+// If the bond is greater than the superbond, it is arbitrated automatically through the EscalationManager; otherwise,
+// it is arbitrated through the DVM.
+contract SuperbondEscalationManager is BaseEscalationManager, Ownable {
+    uint256 public superbond;
```

Comment:
This assumes that requesting contract allows bonding only in one currency. We could either restrict this EM to check `assertion.bond > superbond` only for constant currency or extend this to currency mapping where each configured currency has its own superbond threshold

Reply:
ye, I think for simplicity we hard-code what currency is supported and then enforce that the assertion is only made with that currency.